src/risk_atlas_nexus/ai_risk_ontology/schema/ai_risk.yaml

```diff
@@ -93,7 +94,7 @@ classes:
       - isDefinedByTaxonomy
 
   Action:
-    is_a: Entity
+    is_a: RiskControl
```

Comment:
Hi @ingelise 
Can you exlpain the reason to use Risk Control instead of an Entity for my own information?

Reply:
Thanks @ingelise  for the explanation. So, anything modifies or impact the risk level anyway becomes a subclass of RiskControl, is that right?